app/src/main/java/com/example/androidacademymsk/MockDataSource.kt

```diff
@@ -16,4 +16,16 @@ class MockDataSource {
             MovieCard("Ghost in the Shell"),
         )
     }
+
+    fun getCastCards(): List<CastCard> {
```

Comment:
actually all cast cards should be inside of some movie class, right?

Reply:
I will do it after merge. MockDataSource is now updated in branch master